Makefile

```
@@ -66,3 +66,8 @@ push:
 	$(DOCKER) manifest push --purge $(DOCKER_REPO)/$(DOCKER_IMAGE_NAME):$(VERSION)
 
 push-all: build-all push
+
+.PHONY: docker-repo-name
+docker-repo-name:
+	@echo "$(DOCKER_REPO)/$(DOCKER_IMAGE_NAME)"
```

```suggestion
	@echo "prometheuscommunity/$(DOCKER_IMAGE_NAME)"
```

Otherwise, the output would result into: `ghcr.io/prometheus-community/windows-exporter`

---

This is due to the Makefile including `export DOCKER_REPO ?= ghcr.io/prometheus-community`.

We should probably fix the publishing to default `DOCKER_REPO` to a DockerHub compatible name.

---

I have no clue who decide to push windows_exporter to ghcr only and not following the standards. 

If you agree, I will include the change into the PR #1469 (https://github.com/prometheus-community/windows_exporter/pull/1469/commits/8b6c36bd892f61a727e55238976d839d7f2cac2e)